BrainPortal/app/controllers/groups_controller.rb

```diff
@@ -273,4 +287,11 @@ def group_params #:nodoc:
     end
   end
 
+  def license_check
+    @group = current_user.available_groups.find(params[:id])
+    if current_user.o_unsigned_custom_licenses(@group).present?
+      flash[:error] = "Access to the project #{@group.name} is blocked due to licensing issues."
```

Comment:
For me, it's fine to have this message without having access to the group in CBRAIN. But I am little bit confused to just this kind of message. If I was a user I will not know how to do to resolve this access issue, where I should sign the license?

Reply:
to maintainer or support (added to message)